### PR TITLE
Update translation tooltips and placeholders

### PR DESCRIPTION
**Pull request by stephdl**
This pull request updates the translation tooltips and placeholders in the CreateOrEditHttpRouteModal.vue component. The changes include updating the tooltips for the strip_prefix and url fields, as well as updating the placeholders for the name, host, path, and url fields.

https://github.com/NethServer/dev/issues/6818

image

image
image
image
image
image

### CODE REVIEW

File: core/ui/public/i18n/en/translation.json [hunk 1]
```diff
@@ -608,14 +608,14 @@
         "http_route_details": "HTTP route details",
         "url": "URL",
         "strip_prefix": "Strip URL path prefix",
-        "strip_prefix_tooltip": "E.g.: <code>http://{host}/{path}/{requestpath}?{querystring}</code> is mapped to <code>http://{host}/{requestpath}?{querystring}</code>",
+        "strip_prefix_tooltip": "The prefixes option defines the prefixes to strip from the request URL. E.g.: It allows you to strip a specified prefix from the URL path. If, for example, you remove /api from /api/v1/resource, the resulting path after it is applied would be /v1/resource.",
```
Comment (andre8244) [inline core/ui/public/i18n/en/translation.json:611]: This is not very clear to me: what and where is "prefixes option"?

Comment (stephdl) [reply core/ui/public/i18n/en/translation.json:611]: associate to https://doc.traefik.io/traefik/middlewares/http/stripprefix/

Comment (stephdl) [reply core/ui/public/i18n/en/translation.json:611]: I assume the previous explanation was not much clear hence davidep wanted to enhance it

Comment (andre8244) [reply core/ui/public/i18n/en/translation.json:611]: Then I'd call it "Traefik prefixes option" and add a link to https://doc.traefik.io/traefik/middlewares/http/stripprefix/

**Review by andre8244 — APPROVED** [review]

**Pull request merged**